.github/workflows/ci.yml

```diff
@@ -58,19 +39,10 @@ jobs:
   test-node:
     name: Test node-tests
     runs-on: ubuntu-latest
-    timeout-minutes: 7
+    timeout-minutes: 14
```

Comment:
Hang on, there's an extreme performance regression here. Apparently the node:tests on main take only 6 seconds, and on this branch I'm not even sure this 14 minutes is long enough.

Reply:
(I don't blame pnpm, I blame the fact that this re-rolled the whole lockfile so we got some upgrade that pwned performance.)